DuongHX/Build_System/Install_Nexus/Readme.md

```diff
@@ -0,0 +1,167 @@
+  - ***etc*** thư mục bao gồm những tệp cấu hình.
+  - ***lib*** thư mục này chứa những thư viện nhị phân liên quan đến Apache karaf.
+  - ***public*** thư mục này bao gồm những tài nguyên mở của nexus.
+  - ***system*** thư mục bao gồm tất cả những thành phần và plugin cấu tạo nên nexus.
+
+<a id="data"></a>
+
+### Data directory
+
+  - Thư mục dữ liệu bao gồm tất cả các kho chứa, thành phần và các dữ liệu khác mà được lưu trữ và quản lý bởi ***repository manage***. Thông thường thư mục dữ liệu được lưu trữ ở: ```../sonatype-work/nexus3``` và nó thường được gọi là ```$data-dir``` trong bất kỳ đoạn code nào.
+  - Thư mục dữ liệu bao gồm những thư mục con mà bao gồm những thành phần, kho chứa, cấu hình và các dữ liệu khác.
+  - Những thư mục con bao gồm:
+    - ```Blobs``` : Là thư mục mẹ của tất cả các hệ thống tệp dựa trên blob stores cái mà không được định nghĩa với đường dẫn tuyệt đối.
+    Ví dụ, mặc định blob store thường ở trong ```../sonatype-work/nexus3/blobs/default```.
+    - ```cache``` : Thư mục này bao gồm thông tin về bộ nhớ hiện tại của Karaf bundles.
+    - ```db``` : Thư mục này bao gồm cơ sở dữ liệu OrientDB là bộ lưu trữ chính cho repository quản lý siêu dữ liệu(metadata).
+    - ```elasticsearch``` : Thư mục này chứa trạng thái cấu hình hiện tại của elasticsearch về cơ bản elasticsearch là một công cụ tìm kiếm dựa trên nền tảng Apache Lucene.
+    - ```etc``` : Thư mục này chứa cấu hình thời điểm chạy và tùy chỉnh trình quản lý kho lưu trữ.
+    - ```health-check``` : Thư mục này bao gồm những bộ nhớ về báo cáo nhận được từ tính năng ```Repository Health Check```.
+    - ```keystores``` : Thư mục này chứa những key đươc tạo tự động để  có thể xác định trình quản lý kho lưu trữ.
+    - ```log``` : Thư mục này bao gồm một số tệp log để ghi lại về các khía cạnh khác nhau của trình quản lý kho lưu trữ hiện tại này.
+    - ```tmp``` : Thư mục này được sử dụng để lưu trữ tạm thời.
+
+<a id="run-nexus"></a>
+
+### Run Nexus
+
+- Để chạy nexus chuyển đến thư mục /opt/nexus-3.18.1-01/bin rồi chạy lệnh ```./nexus run``` để chạy nexus, để có thể tắt trình quản lý kho lưu trữ khi đang chạy ta có thể sử dụng tổ hợp phím ```CTR C```.
+- Ngoài ra thay vì run ta còn có thể chạy nexus với các lệnh ```start```, ```stop```, ```restart```, ```force-reload```, ```status``` tương ứng với bắt đầu, dừng, khởi động lại, bắt buộc tải lại hay trạng thái của nexus.
+- Sau nexus bắt đầu chạy ta có thể truy cập thông qua giao diện người dùng bằng cách vào địa chỉ ```http://<server_host>:<port>``` ví dụ ```http://localhost:8081/```.
+![image](images/nexus-home.png)
+- Để có đầy đủ quyền truy cập thì cần phải dùng đến tài khoản admin khi lần đầu đăng nhập ta sẽ thấy thông báo nơi chứa mật khẩu tài khoản admin như sau:
+![image](images/first-login.png)
+
+---------------------------------------------
+
+<a id="run-as-service"></a>
+
+## Run as service. :running_man:
```

Comment:
Theo a nếu dùng tiếng anh thì dùng hết toàn bộ tiếng anh, không để tiêu đề tiếng anh, xong nội dung tiếng việt